templates/charms-architecture.html

```diff
@@ -0,0 +1,191 @@
+{% extends 'how-juju-works.html' %}
+
+{% block title %}Charms architecture{% endblock %}
+{% block meta_keywords %}Software operators{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1nMN5UlwGqI4C4GmYcKy4yRzd9ANUmsVyB1ZcKY7Wpro/edit#heading=h.4tjfso5hx6tw{% endblock meta_copydoc %}
+{% block page_class %}is-paper{% endblock %}
+
+{% set selected_tab='charms-architecture' %}
+
+{% block how_juju_works_content %}
+<section class="p-strip u-no-padding--bottom u-no-padding--top">
+  <div class="row">
```

Comment:
Please use new grid class names where possible (`row--50-50` in this case, or `row--25-75`), with `col` inside them. https://vanillaframework.io/docs/patterns/grid#common-patterns